src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -252,3 +253,28 @@ must be supplied as the value to be matched."
        (sql/do-commands
         (str "ALTER TABLE " table " ALTER COLUMN " column
              " RESTART WITH " restartid))))))
+
+(defn clj->pgobject
```

Comment:
Yeah, I waffled a bit. I put it where I did because the functions originally used postgres? which made the choice easier. My argument for storage_utils is that these functions are ultimately used around postgres/hsql special-casing which I saw mainly in storage_utils, but I can move it if we want.